From 03004fceff4fa0bb4366041e52f64c4c05856ad9 Mon Sep 17 00:00:00 2001
From: Zheng Junjie <z572@z572.online>
Date: Mon, 17 Mar 2025 17:07:28 +0800
Subject: [PATCH] remove werror

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f3e49f7..9c83670 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -31,7 +31,6 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     beluga_compile_options
     INTERFACE -Wall
               -Wconversion
-              -Werror
               -Wextra
               -Wpedantic)
 endif()
-- 
2.47.1

