From 8598dd2c3a29de34d35c2ecc61a6943796d9903b Mon Sep 17 00:00:00 2001
From: Zheng Junjie <z572@z572.online>
Date: Mon, 17 Mar 2025 17:09:38 +0800
Subject: [PATCH] remove werror

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index db241d8..ded1d3f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,6 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     -Wall
     -Wconversion
     -Wextra
-    -Werror
     -Wpedantic)
 endif()
 
-- 
2.47.1

